examples/solvers/adupdates_tomography.py

```diff
@@ -0,0 +1,151 @@
+SPLIT_NUMBER = 20     # How many pieces of data?
+NOISE_LEVEL = 5.0     # How much noise to add to the sinograms
+DATA_FIT_WEIGHT = 1.0  # Regularization parameter (coefficient of the data fit)
+EPSILON = 1e-6        # make sure that we do not divide by zero by adding this
+STEPSIZE = 1.         # Stepsize of the outer iterations
+NITER = 5             # Number of iterations
+RANDOM = True         # Choose the oder of the inner iterations at random?
+
+# --- Create simulated data (phantom) ---
+
+# Reconstruction space: Set of two-dimensional quadratic images with
+# given resolution.
+reconstruction_space = odl.uniform_discr(min_pt=[-DIMENSIONS, -DIMENSIONS],
+                                         max_pt=[DIMENSIONS, DIMENSIONS],
+                                         shape=[RESOLUTION, RESOLUTION])
+phantom = odl.phantom.shepp_logan(reconstruction_space, modified=True)
+
+# Create the forward operators. They correspond to a fully sampled parallel
+# beam geometry.
+geometry = odl.tomo.parallel_beam_geometry(reconstruction_space)
+
+if SPLIT_METHOD == 'block':
+    # Split the data into blocks:
+    # 111 222 333
+    ns = geometry.angles.size // SPLIT_NUMBER
+    ray_trafos = [odl.tomo.RayTransform(reconstruction_space,
+                                        geometry[i * ns:(i + 1) * ns])
+                  for i in range(SPLIT_NUMBER)]
+
+elif SPLIT_METHOD == 'interlaced':
+    # Split the data into slices:
+    # 123 123 123
+    ray_trafos = [odl.tomo.RayTransform(reconstruction_space,
+                                        geometry[i::SPLIT_NUMBER])
+                  for i in range(SPLIT_NUMBER)]
+else:
+    raise ValueError('unknown data split`{}`: typo?'.format(SPLIT_METHOD))
+
+# Create the artificial data.
+data_spaces = [op.range for op in ray_trafos]
+data = [op(phantom) + NOISE_LEVEL * odl.phantom.white_noise(ds)
+        for (op, ds) in zip(ray_trafos, data_spaces)]
+
+# Functionals and operators for the total variation. This is the l1 norm of the
+# (discretized) gradient of the reconstruction. For each of the dimensions
+# we create two functionals and two operators.
+
+# Start with empty lists ...
+tv_functionals = []
+tv_operators = []
+tv_stepsizes = []
+
+# ... and for each dimension of the reconstruction space ...
+reco_shape = reconstruction_space.shape
+reco_dim = len(reco_shape)
+for j in range(reco_dim):
+    # ... add two operators taking only the even and odd elements,
+    # respectively, in that dimension.
+    partial_der = odl.PartialDerivative(
+        reconstruction_space, j, pad_mode='order0')
+    allpoints = list(np.ndindex(reco_shape))
+    sampling_points1 = [list(p) for p in allpoints if p[j] % 2 == 0]
+    sampling_points1 = np.array(sampling_points1).T.tolist()
+    sampling_points2 = [list(p) for p in allpoints if p[j] % 2 == 1]
+    sampling_points2 = np.array(sampling_points2).T.tolist()
+    op1 = 2 * DIMENSIONS / RESOLUTION * odl.SamplingOperator(
```

Comment:
`2 * DIMENSIONS / RESOLUTION` looks very much like `reco_space.cell_sides[0]` to me